CHANGELOG.md

```diff
@@ -7,8 +7,9 @@ NEW FEATURES:
 ENHANCEMENTS:
 * Added `-show-sensitive` flag to tofu plan, apply, state-show and output commands to display sensitive data in output. ([#1554](https://github.com/opentofu/opentofu/pull/1554))
 * Improved performance for large graphs when debug logs are not enabled. ([#1810](https://github.com/opentofu/opentofu/pull/1810))
-* Improved performance for large graphs with many submodules. ([#1809](https://github.com/opentofu/opentofu/pull/1809))
+* Improved performance for large graphs with many subm(odules. ([#1809](https://github.com/opentofu/opentofu/pull/1809))
```

Comment:
Looks like a typo here

Reply:
@cam72cam, I have removed the extra character accidentally included in the commit.